app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ReviewsController < ApplicationController
+  before_action :require_user
+
+  def create
+    @video = Video.find(params[:video_id])
+    @reviews = @video.reviews.all
```

Comment:
Why are you calling `.all` - `@video.reviews` should provide all reviews associated with the `@video`.